### DISCUSSION TITLE
Is that possible to connect muiltiple polts to one full node

### DISCUSSION THREAD

**Discussion by TerryHuang91** [Chia-Network/chia-blockchain#6035] [Support]
I used to created muiltiple full node and ploted a lot of plots within the GUID. Now, I don't want to farm with all those full node. Is that possible to run the full node on one mechine and connect to other farmers. The main point is those plots are created by different account which means the farmer key and pool key are different.

**Comment by XIII39** [comment]
You can setup a single farmer and multiple harvesters. You only need your keys on the farmer machine. Check the Wiki there is a little entry about setting it up. https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

**Comment by room101-dev** [comment]
> I used to created muiltiple full node and ploted a lot of plots within the GUID. Now, I don't want to farm with all those full node. Is that possible to run the full node on one mechine and connect to other farmers. The main point is those plots are created by different account which means the farmer key and pool key are different.

On other machine , you could create as many virtual  env's as you wish, all for different account 'chia keys add ...', then run the farmer on those folders containing the plots. It's best to always use the same plotter, so all your plots have teh same owner, 
I think it would be better to use hpool and setup a worker for each account

... or

In theory you could get a cert ca-key for those other accounts and run a unique env for that instance,

Or better your really just talking about running a pool, where anybody can submit plots and proof of ownership
rather than re-inventing the wheel, why not just use hpool, as what you want is already in place, so long as you can prove ownership of the plots, e.g. provide private-key signature for all plot accounts.

certs are better, then you don't have the all the keys publicly available to hackers

pools are best that way the hackers don't even know your alive, the only IP they have is the pool

...

what you want is 'possible', but not sure you would want to do it, as you would be opening a massive security breach from the node, to the ownership of every single plot in your portfolio, sort of a one stop shopping for the criminal hacker

Like it says above you can put all your keys on one farmer, and then the hacker gets in there and 'chia keys show' and dumps all and then sweeps the funds for all the accounts. Done. Until the there is adequate security in chia to protect keys ( years off ), I think this is a terrible idea.

sheesh, just mine on hpool and let the suckers on chia running solo get robbed, is that where you want to go?

**Comment by BrandtH22** [comment]
@TerryHuang91 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
